src/main/java/hudson/plugins/logparser/LogParserAction.java

```diff
@@ -59,7 +64,7 @@ public static String getUrlNameStat() {
         return urlName;
     }
 
-    public AbstractBuild<?, ?> getOwner() {
+    public Run<?, ?> getOwner() {

```

Comment:
There's an annotation that can be used. It's called `@WithBridgeMethods` or something.